WordPress/src/main/java/org/wordpress/android/util/config/RemoteConfig.kt

```diff
@@ -15,6 +15,8 @@ import org.wordpress.android.util.config.AppConfig.FeatureState
 import javax.inject.Inject
 import javax.inject.Named
 
+const val REMOTE_REFRESH_INTERVAL_IN_HOURS = 12
```

Comment:
@AjeshRPai - a little overkill here, but what if we make TTL a remote value -- could use Firebase for this (defeats the purpose of this project), but would give us the flexibility to adjust. 😜

Reply:
@zwarm Sorry for the late reply. I was focusing on the Hack week and completely ignored the GitHub notifications. 

> defeats the purpose of this project
Yeah, this was my thought as well when I thought we could make this a Remote value. 😄 . Maybe for some time, we can make this a remote value and later on after trial and error we could define a hardcoded time. How does that sound?

Reply:
@AjeshRPai - We don't have to make it a remote value. Let's not overkill - it's my brain working overtime. :)